src/node.cpp

```diff
@@ -1078,6 +1084,18 @@ Node* Node::firstancestor()
     return n;
 }
 
+#ifdef ENABLE_SYNC
+void Node::setSyncable(const bool syncable)
+{
+    mSyncable = syncable;
+}
+
+bool Node::isSyncable() const
+{
+    return mSyncable;
```

Comment:
since the getter/setter functions don't have any side effects, isn't this the same as just having the member public, but with extra steps? ;-)

Reply:
I used setters and getters here because they will have side effects later on in future one-way-sync PRs. So please bare with me :)